### PR TITLE
[Explorer] Fixes flickering between 1M and full number bug

### PR DESCRIPTION
**Pull request by Andrew47**
Currently when an owned coin row is opened, the numbers flick between showing '1M' and '1,000,000'. This PR fixes this issue.

https://user-images.githubusercontent.com/11377188/205658552-0ed5aca3-9d49-41fd-8089-2dd64b89e945.mp4

### CODE REVIEW

File: apps/explorer/src/hooks/useFormatCoin.ts [hunk 1]
```diff
@@ -105,19 +105,19 @@ export function useFormatCoin(
...
 
     const [decimals, queryResult] = useCoinDecimals(coinType);
-    const { isFetched, isError } = queryResult;
+    const { isFetched } = queryResult;
 
     const formatted = useMemo(() => {
         if (typeof balance === 'undefined' || balance === null) return '';
 
-        if (isError) {
+        if (!coinType) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/hooks/useFormatCoin.ts:113]: I imagine we probably want to handle both the error case and the missing coin Type case.

Comment (Andrew47) [reply apps/explorer/src/hooks/useFormatCoin.ts:113]: In the error case, an error message will be printed to the terminal. Is there anything additional that you would like?

Comment (Jordan-Mysten) [reply apps/explorer/src/hooks/useFormatCoin.ts:113]: Actually it looks like the decimals fetching is only enabled based on coin type, so you really shouldn't need this special casing. You probably just need to remove the error case?

Comment (Andrew47) [reply apps/explorer/src/hooks/useFormatCoin.ts:113]: I agree with removing the error case.

**Pull request merged**